src/table/components/__tests__/TableBodyWrapper.spec.jsx

```diff
@@ -44,8 +44,9 @@ describe('<TableBodyWrapper />', () => {
 
   beforeEach(async () => {
     jest.spyOn(selectionsUtils, 'addSelectionListeners').mockImplementation(() => jest.fn());
+    const getLayout = generateLayout(1, 1, 2, [], [{ qText: '100' }]);
```

Comment:
this is not a getter, just call it layout

Reply:
It is because I have another layout at line 60 but I can change it, it will not make any conflict if I call it layout here too.